src/libstd/sys/redox/backtrace/printing/mod.rs

```diff
@@ -0,0 +1,22 @@
+// Copyright 2014-2015 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+pub use self::imp::{foreach_symbol_fileline, resolve_symname};
+
+#[cfg(any(target_os = "macos", target_os = "ios",
+          target_os = "emscripten"))]
+#[path = "dladdr.rs"]
```

Comment:
I guess (multiple) entire files that are useless is probably a waste, so I've pushed a commit addressing this.